dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2869,9 +2855,6 @@ def test_split_row_groups_int_aggregate_files(tmpdir, engine, split_row_groups):
 )
 @pytest.mark.parametrize("split_row_groups", [True, False])
 def test_filter_nulls(tmpdir, filters, op, length, split_row_groups, engine):
-    if engine == "pyarrow" and pa_version < parse_version("6.0.0"):
-        pytest.skip("PyArrow>=6.0.0 needed for null filtering")
```

Comment:
I do think the CI failures are related to https://issues.apache.org/jira/browse/ARROW-15312 (and/or maybe https://issues.apache.org/jira/browse/ARROW-16495). I know it felt nice to remove the version checks, but probably need something like the following for `test_filters_nulls/isna`:

```python
    if engine == "pyarrow" and parse_version(pa.__version__) < parse_version("8.0.0"):
        # See: https://issues.apache.org/jira/browse/ARROW-15312
        pytest.skip("PyArrow>=8.0.0 needed for correct null filtering")
```